parsers/CO.py

```diff
@@ -17,11 +17,19 @@
 from parsers.lib.config import refetch_frequency
 from parsers.lib.exceptions import ParserException
 
+# FIXME: Issues as of 2024-12-29
+#
+# 1. The API isn't accessible from anywhere, for example Sweden can't access it.
+#
+# 2. There is a use of `replace(hour=int(col[-2:]) - 1)`, but should it really
+#    be -1 there?
```

Comment:
Will do a proper review later but the only reason I see for this is if they use a 1 to 24 hour accounting method instead of 0 to 23.

But will try and figure it out later.